medusa/__main__.py

```diff
@@ -1327,7 +1327,7 @@ def start_threads():
                 app.trakt_checker_scheduler.silent = True
             app.trakt_checker_scheduler.start()
 
-            if app.USE_TORRENTS and app.REMOVE_FROM_CLIENT:
+            if app.USE_TORRENTS and app.REMOVE_FROM_CLIENT and not app.TORRENT_METHOD == 'blackhole':
```

Comment:
Instead of `not app.TORRENT_METHOD == 'blackhole'` I'd use `app.TORRENT_METHOD != 'blackhole'`.